### PR TITLE
deps: Update otel 0.61.0

### PR DESCRIPTION
**Pull request by cpheps**
### Proposed Change
- Updated to [v0.61.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.61.0) of OTel
- Still using an updated fork of operations collector but there is a PR out for it to be merged it.
  - GoogleCloudPlatform/opentelemetry-operations-collector#125

##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -430,55 +433,55 @@ require (
...
+	k8s.io/client-go v0.25.2 // indirect
 	k8s.io/klog v1.0.0 // indirect
 	k8s.io/klog/v2 v2.70.1 // indirect
 	k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1 // indirect
-	k8s.io/kubelet v0.25.0 // indirect
+	k8s.io/kubelet v0.25.2 // indirect
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed // indirect
 	sigs.k8s.io/json v0.0.0-20220713155537-f223a00ba0e2 // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect; indirectgithub.com/prometheus/prometheus/model/metadata
```
Comment (BinaryFissionGames) [inline go.mod:479]: What's with this comment here?

File: go.mod [hunk 2]
```diff
@@ -12,136 +12,144 @@ require (
...
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/aerospike/aerospike-client-go/v6 v6.4.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3 // indirect
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl v0.61.0 // indirect
+	github.com/prometheus/prometheus v0.38.0 // indirect
+	github.com/signalfx/signalfx-agent/pkg/apm v0.0.0-20201202163743-65b4fa925fc8 // indirect
+)
```
Comment (BinaryFissionGames) [inline go.mod:136]: I'm curious why this separate block is here?

File: go.mod [hunk 3]
```diff
@@ -12,136 +12,144 @@ require (
...
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/f5cloudexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudpubsubexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlemanagedprometheusexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/influxdbexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/kafkaexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/loadbalancingexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/logzioexporter v0.61.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/lokiexporter v0.60.0
```
Comment (BinaryFissionGames) [inline go.mod:32]: Still v0.60.0?

File: go.mod [hunk 4]
```diff
@@ -12,136 +12,144 @@ require (
...
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.60.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.60.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.60.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.60.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/opencensusexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter v0.61.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/sapmexporter v0.61.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/exporter/signalfxexporter v0.60.0
```
Comment (BinaryFissionGames) [inline go.mod:37]: Same here, still v0.60.0?

File: go.mod (hunk 1 above)
Comment (cpheps) [reply go.mod:479]: I think accidental copy paste

File: go.mod (hunk 2 above)
Comment (cpheps) [reply go.mod:136]: auto generated from `make tidy`. I'll collapse it I prefer just two blocks: direct and indirect

File: go.mod (hunk 3 above)
Comment (cpheps) [reply go.mod:32]: missed it

File: go.mod (hunk 4 above)
Comment (cpheps) [reply go.mod:37]: missed it

**Review by StefanKurek — APPROVED** [review]

**Review by BinaryFissionGames — APPROVED** [review]

**Pull request merged**